File: Extensions/Stateflows.Extensions.PlantUml/Events/PlantUmlRequest.cs
using Stateflows.Common;

namespace Stateflows.Extensions.PlantUml.Events
{
    [NoTracing]
    [NoImplicitInitialization]
    [Event(nameof(PlantUmlRequest))]
    public sealed class PlantUmlRequest : IRequest<PlantUmlInfo>
    { }
}

File: Transport/@stateflows/signalr-client/src/events/plant-uml.request.ts
import { PlantUmlResponse } from "./plant-uml.response";
import { Request } from "./request";

export class PlantUmlRequest extends Request<PlantUmlResponse> {
    public $type = "Stateflows.Extensions.PlantUml.Events.PlantUmlRequest, Stateflows.Extensions.PlantUml";

    constructor() {
        super();
    }
}